ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/evpn-fabric-l3leaf-yml.j2

```diff
@@ -205,9 +225,9 @@
 {%                 set leaf.evpn_rd_admin_subfield = vtep_loopback_network_summary | ipaddr('network') | ipmath(leaf.id + spine.nodes | length) %}
 {%             endif %}
 {%         elif evpn_rd_type.admin_subfield == "leaf_asn" %}
-{%             set leaf.evpn_rd_admin_subfield = leaf.bgp_as %}
+{%             set leaf.evpn_rd_admin_subfield = switch.bgp_as %}
 {%         elif evpn_rd_type.admin_subfield == "spine_asn" %}
-{%             set leaf.evpn_rd_admin_subfield = spine.bgp_as %}
+{%             set leaf.evpn_rd_admin_subfield = switch.bgp_as %}
```

Comment:
I believe this should remain spine.bgp_as

Reply:
Good catch, fixed. Thank you.